server/wages/views.py

```diff
@@ -72,21 +81,32 @@ def get_wages(request):
     return JsonResponse(_get_wages_serialized(request.GET))
 
 
-def _get_wages_serialized(query_params):
+def _get_query_size_limit_to_use(query_params):
     req_limit = _safe_cast(query_params.get("limit"),
                            int,
                            DEFAULT_LIMIT)
     if req_limit < 1:
         req_limit = DEFAULT_LIMIT
     limit = min(req_limit, DEFAULT_LIMIT)
-    query = _construct_wage_query(query_params)
-    paginator = Paginator(query, limit)
+    return limit
+
 
+def _get_query_page_number_to_use(query_params, num_pages):
```

Comment:
Might be sufficient to name the function _get_query_page_number since it is assumed you will be using it.